crates/ruff_python_ast/tests/preorder.rs

```diff
@@ -225,12 +225,6 @@ impl PreorderVisitor<'_> for RecordVisitor {
         self.exit_node();
     }
```

Comment:
Can you add overrides for the new visitor functions or rewrite the test to use `enter_node` and `exit_node`?